api/src/wfl/service/rawls.clj

```diff
@@ -0,0 +1,52 @@
+(ns wfl.service.rawls
+  "Analyze data in Terra using the Rawls API.
+  Note that Firecloud is built on top of Rawls:
+  Rawls may expose functionality that hasn't yet been promoted to Firecloud."
+  (:require [clj-http.client :as http]
+            [clojure.data.json :as json]
+            [clojure.string :as str]
+            [wfl.auth :as auth]
+            [wfl.environment :as env]
+            [wfl.util :as util]))
+
+(defn ^:private rawls-url [& parts]
+  (let [url (util/de-slashify (env/getenv "WFL_RAWLS_URL"))]
+    (str/join "/" (cons url parts))))
```

Comment:
I see this was copied, but do we need to `de-slashify` in either place?
Is there a source for `WFL_RAWLS_URL` that WFL does not control?

Reply:
Speculation only: my understanding of how things work in `environment.clj` tells me that a user could override the value of `WFL_RAWLS_URL` in the command line.